### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by agnosticdev**
Expanded and added a bit more clarity on the installation instruction for CMake on Linux.

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,8 +91,13 @@ cloning over SSH may provide a better experience (which requires
 
 #### CMake
 [CMake](https://cmake.org) is the core infrastructure used to configure builds of
-Swift and its companion projects; at least version 3.4.3 is required. Your
-favorite Linux distribution likely already has a CMake package you can install.
+Swift and its companion projects.  To get started using CMake, at least version 3.4.3 will be required.
```
Comment (CodaFi) [inline README.md]: I think this line was fine the way it was before.

File: README.md [hunk 2]
```diff
@@ -91,8 +91,13 @@ cloning over SSH may provide a better experience (which requires
 
 #### CMake
 [CMake](https://cmake.org) is the core infrastructure used to configure builds of
-Swift and its companion projects; at least version 3.4.3 is required. Your
-favorite Linux distribution likely already has a CMake package you can install.
+Swift and its companion projects.  To get started using CMake, at least version 3.4.3 will be required. 
+Your favorite Linux distribution will most likely already have a CMake package you can install, and was included above in the installation instructions for Linux development dependencies, but in the event CMake is not installed on your Linux system it can be installed with the following command:
+
```
Comment (CodaFi) [inline README.md]: Could you move this below the macOS instructions and make it read like them too?

Comment (agnosticdev) [inline README.md]: No problem at all. This has now been updated.

File: README.md [hunk 3]
```diff
@@ -90,19 +90,19 @@ cloning over SSH may provide a better experience (which requires
...
-```
+[CMake](https://cmake.org) is the core infrastructure used to configure builds of Swift and its companion projects; at least version 3.4.3 is required. 
 
 On macOS, you can download the [CMake Binary Distribution](https://cmake.org/download),
 bundled as an application, copy it to `/Applications`, and add the embedded
 command line tools to your `PATH`:
 
     export PATH=/Applications/CMake.app/Contents/bin:$PATH
+    
+On Linux, if the CMake package was not downloaded in the development dependencies you can download and install the CMake package on the terminal using the following command:
```
Comment (CodaFi) [inline README.md]: Nit: `on the terminal` is redundant

File: README.md [hunk 4]
```diff
@@ -90,19 +90,19 @@ cloning over SSH may provide a better experience (which requires
...
+
+``` 
+sudo apt-get install cmake
+```
```
Comment (CodaFi) [inline README.md]: In general, please wrap like the rest of the document

Comment (agnosticdev) [inline README.md]: No problem.  This has been updated.

**Comment by gottesmm** [comment]
Can you smush these commits together before committing?

**Comment by CodaFi** [comment]
@gottesmm I'll make sure to squash

@swift-ci please smoke test

**Comment by gottesmm** [comment]
Thanks!

**Comment by CodaFi** [comment]
Thanks @agnosticdev!  Welcome to the project.

⛵️

**Comment by agnosticdev** [comment]
Thank you very much, @CodaFi !

**Pull request merged**